src/styles/index.css

```
@@ -0,0 +1,12 @@
+body {
+  margin: 0;
+  font-family: 'roboto', 'Montserrat'
+}
+
+.exercise-header {
+  padding: 1rem 0;
+  margin: 2rem 0;
+  text-align: center;
+  color: #ffffff;
```

Another nit-picky thing, because I'm a total jerk:

```suggestion
  color: #fff;
```

Our CSS guru applicants will probably notice these little things.